lib/pbench/server/api/__init__.py

```diff
@@ -212,7 +212,7 @@ def post(self):
                 response = requests.post(url, json=json_data["payload"])
             else:
                 app.logger.info(
-                    "No playload found in elasricsearch post request json data"
+                    "No playload found in elasticsearch post request json data"
```

Comment:
Technically I think it should still be capitalized; however since this is a log rather than an external message, it's not critical.

Reply:
And for consistency with the message at line 202.